python/stack_of_tasks/ui/model/av_ref.py

```diff
@@ -11,15 +12,20 @@
 
 from . import RawDataRole
 
+icon_path = Path(rospkg.RosPack().get_path("rviz")) / "icons/classes"
+
+
+def get_ipath(icon: str, ext="png") -> str:
+    return str(icon_path / f"{icon}.{ext}")
+
```

Comment:
Isn't that overkill? Doing `QIcon(icon_path / "Axes.png")` below seems clearer (and not longer) to me.